packages/web-react/src/components/Tooltip/__tests__/Tooltip.test.tsx

```diff
@@ -1,42 +1,18 @@
 import '@testing-library/jest-dom';
 import React from 'react';
 import { render, fireEvent } from '@testing-library/react';
-import { classNamePrefixProviderTest } from '../../../../tests/providerTests/classNamePrefixProviderTest';
+// import { classNamePrefixProviderTest } from '../../../../tests/providerTests/classNamePrefixProviderTest';
```

Comment:
Is there any reason why this line is not removed but is commented out?

Reply:
I think it shouldn't be removed and it should be used :)

Reply:
ah, ok, then